modules/core/client/components/Map/osm.json

```diff
@@ -1,6 +1,6 @@
 {
   "version": 8,
-  "name": "OpenStreetMap",
+  "name": "OSM",
```

Comment:
Shorter just fit better in the button without going to multiple lines. 😁 it's for internal use only anyway.